thirdparty_containers/system-test/playlist.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>mauve-system-test</testCaseName>
+		<variations>
+			<variation>-XX:+UseContainerSupport</variation>
```

Comment:
Given this is an openj9 only option, another reason to explicitly state the `<impls><impl>openj9</impl></impls>` in this `<test>` block, otherwise the underlying mauve tests may try to execute against hotspot and fail

Reply:
-XX:+UseContainerSupport - this is turned on by default now, will remove this variation